src/components/toggle-group/_editor.scss

```diff
@@ -1,12 +1,20 @@
+.components-toggle-group {
+	overflow: hidden;
```

Comment:
I think the animation would look better with with `overflow: visible`. Now the item it looks like is cut-off.
This is how it looks with `overflow: hidden`: https://share.getcloudapp.com/yAu27wmp
And this is how it looks with `overflow: visible`: https://share.getcloudapp.com/jkuld9GP

Reply:
@madalingorbanescu I'm not seeing much a difference as there is a "jump" throughout the recording video – is that caused by the overflow or merely by scroll?

Reply:
That jump was caused by the scroll.
Here is another videos, where the difference is more visible:
With `overflow: hidden`: https://share.getcloudapp.com/eDu6QmkO
With `overflow: visible`: https://share.getcloudapp.com/yAu27pXW

When an element is removed, and added in **Elements you aren't using**, the right border on that element is not visible until the transition ends.

Reply:
@madalingorbanescu I get it now – good catch! 👏 

Small but nice improvement – it seems like we're polishing a diamond here 💎😊